time_calculator.rb

```diff
@@ -0,0 +1,20 @@
+module TimeCalculator
+
+  def time  () Time.now   end
```

Comment:
def time
 Time.now
end

We are paying you to write understandable code. Shaane banne ke liye nahi.

Reply:
😅